src/tactic/clear.lean

```diff
@@ -83,7 +82,7 @@ end
 -/
 meta def clear' (p : parse (many ident)) : tactic unit := do
 hyps ← p.mmap get_local,
-tactic.clear' false (rb_map.set_of_list hyps)
+tactic.clear' false hyps
```

Comment:
Will this now fail if you call `clear' a a a a a`?  Not that you should, I'm just trying to figure out why there was an `expr_set` in the first place.

Reply:
`tactic.clear` had a bug that caused a segfault if you do this. It might be fixed now

Reply:
It's been fixed: both `clear' a a` and `clear_dependent a a` give the reasonable error message:
`revert tactic failed, duplicate hypothesis 'a'`. I've added a test to this effect.